examples/metrics/kube-state-metrics/configmap.yaml

```diff
@@ -205,3 +205,21 @@ data:
                 connectorPluginsVersion: [ status, connectorPlugins, version ]
                 labelSelector: [ status, labelSelector ]
                 replicas: [ status, replicas ]
+        - groupVersionKind:
+            group: access.strimzi.io
+            version: v1alpha1
+            kind: KafkaAccess
```

Comment:
What happens when `KafkaAccess` is not installed? As many users will not be using the Access Operator and thus not have the CRD present.